modules/ocf_kubernetes/templates/master/loadbalancer/haproxy.cfg.erb

```diff
@@ -28,7 +28,7 @@ frontend lb-http
   redirect scheme https code 301 if !{ ssl_fc }
 
 frontend lb-https
-  bind 0.0.0.0:443 ssl crt /etc/ssl/private/<%= @fqdn %>.pem
+  bind 0.0.0.0:443 ssl crt /etc/ssl/private/lb.ocf.berkeley.edu.pem
```

Comment:
I see why you did this, and yeah it makes sense. This is strictly clearer than fqdn, but would slightly prefer a constant. If you left as-is that's OK too.

Reply:
I'm planning to revamp this config heavily at some point in the future as well. There are a lot of things that can be improved.